src/media/css/app-list-filters.styl

```diff
@@ -75,6 +76,10 @@ App listing filters.
     margin-left: 8px;
     width: 84px;
 
+    +rtl() {
+        float: left;
```

Comment:
This is an inner child of a different parent. I would've converted the full structure to flexbox but it really doesn't make sense here and is only used for the mobile subheader where an app list toggle is present.